platform-sdk/platform-apps/tests/StatsSigningTestingTool/src/main/java/com/swirlds/demo/stats/signing/StatsSigningTestingToolMain.java

```diff
@@ -282,7 +282,7 @@ private synchronized void generateTransactions() {
 
     @Override
     public SwirldState newState() {
-        return new StatsSigningTestingToolState(selfId, () -> transactionPool);
```

Comment:
can `selfId` be removed from this class as well?

Reply:
removed